### PR TITLE
Kitbag medic retextures, simple medic patches & minotaur redo

### PR DESCRIPTION
**Pull request by JoramD0**
**When merged this pull request will:**
- Add white kitbag variant
- Add a medic version to all kitbag camo variants
- Add medic shoulder patches
- Include redo of the minotaur patches

### CODE REVIEW

**Comment by JoramD0** [comment]
Testing out a few changes/fixes for the patches to try and eliminate the weird border on some uniforms.

File: addons/insignias/CfgUnitInsignia.hpp [hunk 1]
```diff
@@ -711,4 +711,40 @@ class CfgUnitInsignia {
         displayName = "Contractor Badass";
         texture = QPATHTOF(data\contractorbadass.paa);
     };
+    class CLASS(Insignia_MedicSimple_Black) {
+        MACRO_INSIGNIA_COMMON
+        author = "Gilleedoo, JoramD";
+        displayName = "Simple Medic (Black)";
+        texture = QPATHTOF(data\medicSimple_blk.paa);
```
Comment (jonpas) [inline addons/insignias/CfgUnitInsignia.hpp]: `snake_case` please.

File: addons/insignias/CfgUnitInsignia.hpp [hunk 2]
```diff
@@ -711,4 +711,40 @@ class CfgUnitInsignia {
         displayName = "Contractor Badass";
         texture = QPATHTOF(data\contractorbadass.paa);
     };
+    class CLASS(Insignia_MedicSimple_Black) {
+        MACRO_INSIGNIA_COMMON
+        author = "Gilleedoo, JoramD";
```
Comment (jonpas) [inline addons/insignias/CfgUnitInsignia.hpp]: He is named `GilleeDoo` (capital `D`) in previously contributed classes.

File: addons/backpacks/CfgVehicles.hpp [hunk 3]
```diff
@@ -69,6 +69,104 @@ class CfgVehicles {
...
+            QPATHTOF(data\backpack_kitbag_medic_white_co.paa)
+        };
+    };
+
+    class CLASS(Backpack_Kitbag_Medic_Black): B_Kitbag_Base {
+        dlc = QUOTE(PREFIX);
+        scope = 2;
+        author = "Bohemia Interactive, GilleeDoo, JoramD";
+        displayName = CSTRING(Backpack_Kitbag_Medic_Black);
+        picture = QPATHTOF(UI\backpack_kitbag_medic_blk_ca.paa);
```
Comment (jonpas) [inline addons/backpacks/CfgVehicles.hpp]: Can we do naming of those files in full, like the rest?

File: addons/backpacks/CfgVehicles.hpp [hunk 4]
```diff
@@ -69,6 +69,104 @@ class CfgVehicles {
...
+        scope = 2;
+        author = "Bohemia Interactive, GilleeDoo, JoramD";
+        displayName = CSTRING(Backpack_Kitbag_Medic_Coyote);
+        picture = QPATHTOF(UI\backpack_kitbag_medic_coyote_ca.paa);
+        hiddenSelectionsTextures[] = {
+            QPATHTOF(data\backpack_kitbag_medic_coyote_co.paa)
+        };
+    };
+
+    class CLASS(Backpack_Kitbag_Medic_Digi): B_Kitbag_Base {
```
Comment (jonpas) [inline addons/backpacks/CfgVehicles.hpp]: `Digi` -> `Digital` everywhere.

**Pull request merged**